examples/js/algolia-experiences/package.json

```diff
@@ -8,6 +8,10 @@
     "website:examples": "BABEL_ENV=parcel parcel build *.html --public-url . --dist-dir=../../../website/examples/js/algolia-experiences"
   },
   "browserslist": "firefox 68, chrome 78, IE 11",
+  "dependencies": {
```

Comment:
Technically these are only required when using the `local.html` file, but unsure how else to get the build dependencies fixed otherwise.

Reply:
yeah good call